runtime/tutor.txt

```diff
@@ -43,15 +43,15 @@ _________________________________________________________________
 =                        EXITING HELIX                          =
 =================================================================
 
- 1. Press the : key to enter command mode. Your cursor will
+ 1. Press the : key to enter Command Mode. Your cursor will
```

Comment:
I'd rather see it either be "Command mode" or "command mode".

Reply:
Thank you for the review. I am not attached to this, but hoped to bring more clarity between the abstract mention of a mode and a specific mention of a mode, for example "Command Mode," by following 'proper noun' rules of english capitalization.

For example, if someone wrote "New York Cathedral" they would capitalize Cathedral rather than "New York cathedral" as part of the 'proper noun' even though 'cathedral' is not generally capitalized, and there are other cathedrals that exist.

Between the two preferences you expressed I would think the all lower case is better, because it is not capitalizing part but not all of the specific mode in question seems to go against this rule, whether  or not it is a proper noun.

In the end, I am just aiming for the brand new user to have more ability to understand modes if they are not familiar with the concept.